testcase.py

```diff
@@ -1327,6 +1330,44 @@ def _get_full_current_state(self):
 
         return z
 
+    def store_results(self):
+        '''Stores results from scenario in working directory.
+        When ran with service, the result will be packed in the result tarball and
+        be retrieveable with the test_id
+
+
+        Returns
+        -------
+        None.
+
+        '''
+
+        name = "results"
+        # get results_json
```

Comment:
To reduce code duplication, I suggest abstracting the creation of a results.json structure in a separate function for all fields that are common with what is submitted to dashboard, defined here https://github.com/ibpsa/project1-boptest/blob/master/testcase.py#L1143.  Then, ``post_results_to_dashboard`` and ``store_results`` can share use of that function to build an initial json and add additional fields as-needed  (if any) for submission to dashboard or writing to file, respectively.

Reply:
I proposed a solution. However, I had some trouble with submitting results to the dashboard, with both previous and updated solution, so I have not checked that it works properly.

Reply:
@HWalnum I'll take a look, but looks like you got a couple results submitted for BESTEST Hydronic Heat Pump under the account "hwaln" at https://dashboard.dev.boptest.net/.

Reply:
Thanks @dhblum. I think those are from some extra tests I did with the service.

Reply:
Ok actually I'm having some trouble submitting to the dashboard too.

Reply:
And Ok I see.  I'm meeting with Kyle tomorrow morning for a few things and he can help work out what's going on.